src/serverroutes.js

```diff
@@ -142,6 +143,11 @@ module.exports = function(app, saveSecurityConfig, getSecurityConfig) {
 
   app.put(`${SERVERROUTESPREFIX}/security/config`, (req, res, next) => {
     if (app.securityStrategy.allowConfigure(req)) {
+      if (!ms(req.body.expiration)) {
```

Comment:
With this change serverroutes becomes dependent on how tokensecurity works internally. What if we offload validating of the config is ok to tokensecurity, wholesale? Then tokensecurity can validate `expiration` by trying the exact same signing operation where it will be eventually used, so that even the `ms` dependency is not direct and if `jsonwebtoken`  switches to the next `ms` version or something the validation will always be in sync. `ms` used here is not necessarily the same `ms`  that jsonwebtoken uses.

A bit theoretical, but still.